apps/template/mix.exs

```diff
@@ -29,7 +29,7 @@ defmodule Template.MixProject do
   defp deps do
     [
       {:brook, "~> 0.4"},
-      {:divo, "~> 1.1", only: [:dev, :test, :integration]},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in template, template needs a new mix version